### ISSUE TITLE
D401 does not properly check one-word documentations

### ISSUE THREAD

**Issue by Strus** [PyCQA/pydocstyle#421]
pydocstyle version: 4.0.1

D401 does not detect errors in documentation if it is just one word.

Example:

```python
"""Test file."""

class Foo:
    """Represents something."""

    def __init__(self, bar):
        """Constructor.

        :param bar: bar
        """
        self.bar = bar
```

```bash
pydocstyle --ignore D203,D213,D407,D416 test.py
<empty output>
```

When I add some more words to `__init__` documentation, it properly detects D401:

```python
"""Test file."""

class Foo:
    """Represents something."""

    def __init__(self, bar):
        """Constructor of Foo.

        :param bar: bar
        """
        self.bar = bar
```

```bash
pydocstyle --ignore D203,D213,D407,D416 test.py
test.py:7 in public method `__init__`:
        D401: First line should be in imperative mood; try rephrasing (found 'Constructor')
```

It also does not detect other single words like for example `Initalizer.`
